src/main/java/org/jenkinsci/plugins/workflow/actions/PersistentAction.java

```diff
@@ -0,0 +1,10 @@
+package org.jenkinsci.plugins.workflow.actions;
+
+import hudson.model.Action;
+
+/**
+ * This is a marker interface for an action that can't be contributed by a {@link jenkins.model.TransientActionFactory}.
+ * Actions implementing this can use more efficient {@link hudson.model.Actionable#getAction(Class)} variant
```

Comment:
Feel free to `@link` to `getPersistentAction`—its being `@Restricted` does not prevent it from appearing in Javadoc.